src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -841,6 +843,229 @@ def test_WINNF_FT_S_SIQ_11(self):
+
+    # Step4: Load N3 PPA
+    for pal_record in config['palRecordsN3']:
+      self._sas_admin.InjectPalDatabaseRecord(pal_record[0])
+    for ppa_record in config['ppaRecordsN3']:
+     zone_id = self._sas_admin.InjectZoneData({"record": ppa_record})
+
+ 
+    # Step5: Trigger CPAS activity
+    if (len(config['gwpzRecordsN1']) > 0 or len(config['palRecordsN3']) > 0):
+      self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete() 
+
+    # Step 6: Send N2 spectrum inquiry requests (one per registered CBSD)
+    spectrum_inquiry_list = config['spectrumInquiryRequestsN2']
+    addCbsdIdsToRequests(cbsd_ids, spectrum_inquiry_list)
+ 
+    request = {'spectrumInquiryRequest': spectrum_inquiry_list}
+    response = self._sas.SpectrumInquiry(request)['spectrumInquiryResponse']
+
+    # Check the response contains N2 SpectrumInquiryResponse objects
+    self.assertEqual(len(response),len(config['spectrumInquiryRequestsN2']))
+
+    for index,response in enumerate(response):
+
+       # If the corresponding request contained a valid cbsdId, the
+       # response shall contain the same cbsdId.
+       if 'cbsdId' in spectrum_inquiry_list[index]:
+        if spectrum_inquiry_list[index]['cbsdId'] in cbsd_ids:
+          self.assertEqual(response['cbsdId'], spectrum_inquiry_list[index]['cbsdId'])
+        else:
+          self.assertFalse('cbsdId' in response)
+
+       # Check response code matches the expected response code
+       self.assertIn(response['response']['responseCode'],config['expectedResponseCodes'][index])
+       if (response['response']['responseCode'] == 0) :
+
+          # Check if the 'availableChannel' parameter is present
+          self.assertTrue('availableChannel' in response)
+
+          # Check if the ruleApplied field is set to FCC_PART_96
+          for channel in response['availableChannel']:
+            self.assertEqual(channel['ruleApplied'], 'FCC_PART_96')
+
+          # Check that the GAA channel frequency ranges do not overlap with FR1cbsd ranges
+          for channel in response['availableChannel']:
+            if channel['channelType'] == 'GAA':
+                for fr1_range in config['fr1Cbsd'][index]:
+                   
+                   # Check if the channel range is lower than the low freq of FR1cbsd range or if it is higher 
+                   # than the high freq. If not then it is considered to be overlapping
+                   if ((channel['frequencyRange']['lowFrequency'] < fr1_range['frequency']['lowFrequency']) and 
+                       (channel['frequencyRange']['highFrequency'] <= fr1_range['frequency']['lowFrequency'])):
+                      pass
+                   else:
+                      if  ((channel['frequencyRange']['lowFrequency'] >= fr1_range['frequency']['highFrequency']) and 
+                       (channel['frequencyRange']['highFrequency'] > fr1_range['frequency']['highFrequency'])):
+                          pass 
+                      else:
```

Comment:
Could we use if / elif /else instead of having these over-indented conditions?

Reply:
Modified   the statements to assertTrue